seed/loan.sql

```diff
@@ -0,0 +1,1006 @@
+  (639, 718, '10/01/2006', '08/14/2018', '03/20/2020', '03/05/2014', '07/26/2005'),
+  (805, 876, '10/25/2005', '10/23/2011', '09/22/2021', '07/24/2020', '10/29/2022'),
+  (479, 671, '04/23/2021', '08/28/2010', '11/28/2010', '04/09/2023', '10/25/2009'),
+  (259, 826, '04/26/2017', '05/29/2022', '12/24/2022', '08/31/2017', '12/22/2014'),
+  (457, 394, '05/01/2021', '05/13/2023', '05/01/2010', '04/10/2023', '07/24/2016'),
+  (303, 150, '03/15/2017', '05/19/2013', '10/19/2009', '02/23/2012', '07/07/2015'),
+  (158, 381, '02/22/2008', '07/07/2012', '07/30/2006', '04/07/2019', '08/09/2022'),
+  (847, 660, '04/23/2018', '04/28/2013', '12/02/2007', '04/12/2009', '04/13/2015'),
+  (121, 742, '08/27/2009', '05/08/2011', '11/11/2011', '01/07/2013', '09/10/2017'),
+  (953, 552, '09/07/2011', '05/06/2012', '05/20/2023', '01/10/2013', '09/17/2022'),
+  (715, 478, '04/28/2008', '12/13/2010', '03/25/2005', '05/02/2012', '05/03/2010'),
+  (180, 58, '06/15/2018', '06/16/2009', '12/12/2010', '04/05/2012', '12/18/2021'),
+  (872, 161, '06/11/2014', '06/15/2006', '12/13/2007', '09/09/2014', '02/05/2014'),
+  (956, 96, '07/24/2009', '07/05/2014', '03/29/2012', '06/03/2005', '04/03/2008'),
+  (571, 946, '01/07/2005', '08/23/2016', '10/13/2019', '10/28/2018', '03/15/2018'),
+  (206, 505, '12/13/2017', '10/21/2012', '02/22/2017', '12/17/2013', '02/07/2022'),
+  (226, 88, '03/14/2016', '06/12/2016', '05/15/2007', '12/17/2021', '08/09/2021'),
+  (22, 122, '03/11/2015', '01/29/2022', '02/15/2020', '07/26/2013', '08/27/2013'),
+  (289, 260, '11/06/2014', '12/03/2018', '03/15/2016', '10/14/2011', '08/29/2019'),
+  (647, 242, '06/20/2018', '04/25/2016', '12/12/2006', '12/21/2021', '01/28/2005'),
+  (318, 149, '02/04/2021', '09/12/2008', '09/26/2006', '03/10/2019', '04/26/2007'),
+  (867, 122, '02/10/2020', '02/26/2006', '03/21/2013', '09/14/2019', '05/14/2020'),
+  (844, 363, '06/15/2017', '09/20/2017', '07/16/2018', '04/08/2018', '09/21/2010'),
+  (290, 5, '12/22/2011', '01/09/2015', '06/04/2020', '04/05/2020', '12/26/2018'),
+  (465, 985, '02/17/2006', '06/29/2023', '06/03/2018', '03/25/2015', '01/13/2016'),
+  (554, 395, '02/05/2023', '01/06/2018', '12/19/2009', '07/21/2017', '05/17/2008'),
+  (314, 43, '09/21/2009', '05/27/2009', '05/09/2022', '10/15/2011', '01/13/2005'),
+  (461, 268, '11/27/2022', '01/25/2021', '07/02/2016', '06/26/2018', '08/27/2022'),
+  (202, 663, '02/17/2015', '05/12/2005', '10/05/2018', '06/15/2021', '10/29/2007'),
+  (199, 456, '11/06/2016', '02/12/2022', '05/06/2010', '01/17/2016', '04/06/2005'),
+  (896, 417, '01/30/2006', '08/01/2020', '05/19/2005', '07/30/2008', '01/01/2008'),
+  (380, 423, '06/24/2013', '10/12/2012', '09/04/2010', '10/29/2005', '01/07/2022'),
+  (728, 260, '06/02/2015', '08/26/2006', '12/17/2008', '11/09/2011', '11/03/2009'),
+  (992, 695, '10/31/2008', '07/27/2005', '01/30/2019', '02/11/2014', '05/28/2019'),
+  (531, 564, '06/30/2007', '02/14/2017', '04/07/2018', '07/16/2021', '12/11/2009');
```

Comment:
add GO at the end of each insert script 
and you can combine them into one scrip that way

Reply:
Wouldn't that affect readability?

Reply:
its a dummy data anyway we dont need 3 scripts for it and if we keep them in one file it will be easier to review them later on if some data if corrupter of something.

Reply:
change the files names to indicate that they are seeding data